tests/test_app.py

```diff
@@ -51,6 +51,27 @@ def test_metrics(client, now, enrollment):
         match_querystring=True,
         json=sample_data
     )
+    metric_name = b'my_costs'
+    app.config['METRIC_NAME'] = metric_name
+
+    rsp = client.get('/metrics')
+    assert rsp.status_code == 200
+    assert rsp.data.count(metric_name) == 4
+
+
+@responses.activate
+def test_metrics_name_default_value(client, now, enrollment):
+    responses.add(
+        method='GET',
+        url="https://ea.azure.com/rest/{0}/usage-report?month={1}&type=detail&fmt=Json".format(enrollment, now),
+        match_querystring=True,
+        json=sample_data
+    )
+    #make sure that app.config is empty
```

Comment:
Actually that was a difference between py2 and py3 that triggered this messy cleanup code.

A better way would probably be to reset `app.config` in the fixture.